test/Unosquare.Swan.Test/ObjectComparerTest.cs

```diff
@@ -82,5 +82,66 @@ public void CompareEqualArrayWithObjects()
 
             Assert.IsTrue(ObjectComparer.AreEnumsEqual(first, second));
         }
+
+        public enum aux1
+        {
+            one, two, three
+        }
+
+        public enum aux2
+        {
+            one, two, three
+        }
+        
+        [Test]
+        public void AreEnumsEqualLengthTest()
+        {
+            List<string> leftListEnum = new List<string>()
+            {
+                "ArCiGo",
+                "ElCiGo",
+                "WizardexC137",
+                "DCOW"
+            };
+
+            List<string> rightListEnum = new List<string>()
+            {
+                "Kadosh"
+            };
+
+            Assert.IsFalse(ObjectComparer.AreEnumsEqual(leftListEnum.AsEnumerable(), rightListEnum.AsEnumerable()));
+        }
+        
+        [Test]
+        public void AreObjectsEqualsFalse()
+        {
+            List<object> leftObject = new List<object>()
+            {
+                "ArCiGo",
+                "ElCiGo",
+                "ArCiNa"
+            };
+
+            List<object> rightObject = new List<object>()
+            {
+                "Elsa",
+                "Mariana",
+                "Alejandro",
+                "Nestor",
+                "Grecia"
+            };
+
+            Assert.IsFalse(ObjectComparer.AreEnumsEqual(leftObject, rightObject));
```

Comment:
public void AreObjectsEqualsFalse()...Objects or Enums?

Reply:
@buda1994

Reply:
@geoperez That method is mine, jajajaja. I changed it